Doc/whatsnew/3.13.rst

```diff
@@ -1566,428 +1851,92 @@ although there is currently no date scheduled for their removal.
   use :meth:`~zipimport.zipimporter.exec_module` instead.
 
 
-Removed
-=======
+CPython Bytecode Changes
```

Comment:
Surely there are more changes to this type of bytecode details. Maybe Irit can help auditing this.